src/Cohesity.Powershell/Scripts/Recover-CohesityBackupToView.ps1

```diff
@@ -112,6 +108,4 @@ function Recover-CohesityBackupToView {
     }
     End {
     }
-}
-
-Recover-CohesityBackupToView -QOSPolicy "TestAndDev High"
```

Comment:
remove all trailing whitespaces chars in all files

Reply:
No files with trailing spaces